vagrant/boxes.d/00-centos.yaml

```diff
@@ -9,3 +9,8 @@ centos7:
 centos7-fips:
   box_name: 'pulp/centos7-fips'
   pty: true
+
+centos8-stream:
+  box_name:   'centos/8-stream'
+  box_url:    'https://cloud.centos.org/centos/8-stream/x86_64/images/CentOS-Stream-Vagrant-8-20200113.0.x86_64.vagrant-libvirt.box'
```

Comment:
Looks like the version # is hardcoded into the URL, is there any way to avoid that?

Reply:
Based on my research, no. I'll ask though.

Reply:
CentOS devs say they are working on that.

Reply:
Also, once CentOS 8.2 goes GA, we will not need stream as badly: https://pulp.plan.io/issues/6304